packages/wp-story-editor/src/components/fontCheck/fontCheckDialog.js

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { TranslateWithMarkup, __ } from '@googleforcreators/i18n';
+import { trackClick } from '@googleforcreators/tracking';
+import styled from 'styled-components';
+import { useState, useCallback } from '@googleforcreators/react';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  Text,
+  THEME_CONSTANTS,
+  Icons,
+  Link,
+} from '@googleforcreators/design-system';
+
+import {
+  useStory,
+  useConfig,
+  Dialog,
+  FontPicker,
+  TEXT_ELEMENT_DEFAULT_FONT,
+} from '@googleforcreators/story-editor';
+
+const DialogContent = styled.div`
+  display: grid;
+  row-gap: 21px;
```

Comment:
How did you land on 21 here? Typically we like to do things divisible by 4, could it be 20 instead?

Reply:
Will make it 20  --- this tried to measure via the Figma file.

Reply:
Updated here https://github.com/GoogleForCreators/web-stories-wp/commit/59fd388d1c46eb9058249c90a8981fd8d078f71c#diff-c872631c2fd1bbe33e1710d4fa80f88e897936663ef1192d56f0df781ea31da5R44